scripts/conveyorBelt.js

```diff
@@ -0,0 +1,16 @@
+class ConveyorBelt extends Cube {
+  constructor(x, y, name, direction) {
+    super(x, y, AssetStore.getTexture(`conveyorBelt${direction}`));
+    this.direction = direction;
+  }
+  onCollision(collisionSide) {
+    if (collisionSide === "bottom") {
```

Comment:
Since you only care about the value of the `collisionSide` variable I would use a switch here. A switch tells you just that.

```javascript
switch (collisionSide) {
  case "bottom":
    // Do stuff.
  // ...  
}
```

Reply:
Yep, I was just lazy. I think a have another, sorter solution for this thing though, but I have not yet tried it :p

Reply:
This is done.